### PR TITLE
Update steps for creating a local/remote file repository

### PR DESCRIPTION
**Pull request by dubewarsagar**
Added missing steps required for creating a local/remote file type repo. 
This is a copy of PR #1650.
This PR contains steps for EL 7 and EL 8.
This PR should be merged in foreman-3.1 only.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_creating-a-remote-file-repository.adoc [hunk 1]
```diff
@@ -25,24 +25,40 @@ endif::[]
 
 To create a file type repository in a remote directory, complete the following procedure:
 
-. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. On your machine, ensure that the right repositories are enabled.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-remote-file-repository.adoc:29]: Maybe reword "right repositories"?

File: guides/common/modules/proc_creating-a-remote-file-repository.adoc [hunk 2]
```diff
@@ -25,24 +25,40 @@ endif::[]
...
 ----
-# yum install python3-pulp_manifest
+# {package-install} python3-pulp_manifest
 ----
-. Create a directory that you want to use as the file type repository in the HTTP server's public folder:
+** For {EL} 7:
 +
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum tfm-pulpcore-python3-pulp_manifest
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-remote-file-repository.adoc:57]:
```suggestion
# yum install tfm-pulpcore-python3-pulp_manifest
```

Also, I am unsure about the package name. Maybe someone else can confirm?

File: guides/common/modules/proc_creating-a-remote-file-repository.adoc [hunk 3]
```diff
@@ -25,24 +25,40 @@ endif::[]
...
-. Create a directory that you want to use as the file type repository in the HTTP server's public folder:
+** For {EL} 7:
 +
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum install tfm-pulpcore-python3-pulp_manifest
+----
+. Create a directory that you want to use as the file type repository in the HTTP server's public folder:
 [options="nowrap" subs="+quotes"]
++
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-remote-file-repository.adoc:61]:
```suggestion
+
[options="nowrap" subs="+quotes"]
```

File: guides/common/modules/proc_creating-a-local-file-repository.adoc [hunk 4]
```diff
@@ -27,6 +28,18 @@ endif::[]
...
 ----
+ifdef::satellite[]
++
+Note that this command stops the {Project} service and re-runs {foreman-installer}.
+Alternatively, to prevent downtime caused by stopping the service, you can use the following:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages unlock
+# {package-install} python3-pulp_manifest
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: again, I am unsure about the package name. Could you please provide a link to yum.theforeman.org for this package? Or can someone else confirm?

Comment (dubewarsagar) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: @lpramuk, could you please help here? 
Thanks!

Comment (ekohl) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: It is indeed a problem that it isn't published in upstream. That was intended to be fixed. It's a very valid point because we do have packages with a `python38` and `python39` prefix so it's quite crucial to get it right.

Comment (dubewarsagar) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: I'm not sure what fix should be here in that case, Can someone guide me here if anything is pending from my side? Thanks!

Comment (maximiliankolb) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: I've opened an issue against pulpcore-packaging: https://github.com/theforeman/pulpcore-packaging/issues/598

My understanding is that once the Pipeline ran through, the built package will be available at http://yum.theforeman.org/pulpcore/ where you can get the package name from.

Comment (Lennonka) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc:39]: @dubewarsagar Pls, correct the package names in both modules. The correct package names for 6.11 are:

- on EL7: `tfm-pulpcore-python3-pulp_manifest`
- on EL8: `python38-pulp_manifest`

(I checked in Ohsnap and I'm assuming it corresponds with pulpcore 3.16)

Comment (dubewarsagar) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc]: Updated package names in both modules.

Comment (dubewarsagar) [reply guides/common/modules/proc_creating-a-local-file-repository.adoc]: @Lennonka updated package names in both the modules. 
@maximiliankolb pls check now and in case it is good to merge, kindly do the needful. 

Thank you both :-)
